app/controllers/api/v2/job_invocations_controller.rb

```diff
@@ -18,6 +18,7 @@ def index
 
       api :GET, '/job_invocations/:id', N_('Show job invocation')
       param :id, :identifier, :required => true
+      param :host_status, String, required: false, allow_blank: true, desc: N_('Show Job status for the hosts.')
```

Comment:
Do we actually need this? I mean, it's just a string in the response. I think it could be just added.

Reply:
Yop we need it, because of this: https://projects.theforeman.org/issues/28723.

When non-admin user is getting data for too many hosts, checking permissions and getting data takes a lot of time.